packages/react-components/src/components/styles/LoadingSpinner.style.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { mergeStyles } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const loadingSpinnerContainerStyle = (): string =>
+  mergeStyles({
+    width: '100px',
```

Comment:
use `rem` instead of pixles throughout (can manually convert (divide by 16) or use the _pxToRem fn we have)